app/admin/practices.rb

```diff
@@ -5,14 +5,110 @@
 
   scope :published
   scope :unpublished
+  scope :Get_Practice_Owner_Emails
+
+  csv do
+    if params[:scope] == "get_practice_owner_emails"
+      column(:practice_name) {|practice| practice.name}
+      column(:owner_email) {|practice| practice.user&.email}
+    else
+      column :id
```

Comment:
wondering if we can do something fancy here so we don't have to update it when we add columns...

Reply:
Yeah - I was looking for a way to default to all columns in that else case - ie.., columns.* or something - but didn't find one.   I can do some more research on that today.

Reply:
I figured out how to add the column names dynamically - so for published and unpublished scopes/filters.. all column names in the Practice Table will be included in the csv.  Code has been pushed to server for dm-1687